public/data/vivekpal1.json

```
@@ -0,0 +1,68 @@
+{
+  "name": "Vivek Pal",
+  "type": "personal",
+  "bio": "Full Stack Deveoper. Learning Web3. Open Source Enthusiast! Founder and Developer Shastra OS",
+  "avatar": "https://avatars.githubusercontent.com/u/88021457?v=4",
```

I recommend using the more dynamic link for your profile because that will get automatically updated when you change your avatar + it's way shorter! 
```suggestion
  "avatar": "https://github.com/vivekpal1.png",
```

---

Done, thanks for Suggesting